analytics-api/src/analytics_api/models/request_type_option.py

```diff
@@ -28,15 +29,26 @@ def get_survey_result(
                                .subquery())
 
         # Get all the survey questions specific to a survey id which are in active status.
-        survey_question = (db.session.query(RequestTypeOption.position.label('position'),
-                                            RequestTypeOption.label.label('label'),
-                                            RequestTypeOption.request_id)
-                           .filter(and_(RequestTypeOption.survey_id.in_(analytics_survey_id),
-                                        RequestTypeOption.is_active == true(),
-                                        or_(RequestTypeOption.display == true(),
-                                            RequestTypeOption.display.is_(None))))
-                           .order_by(RequestTypeOption.position)
-                           .subquery())
+        # for users with role to view all surveys fetch all survey questions
+        # for all other users exclude questions excluded on report settings
+        if can_view_all_survey_results:
```

Comment:
is it at all possible to create can_view_all_survey_results variable in same file where we need it? instead of passing it on